frontend/src/pages/MyCourses/MyCourses.tsx

```diff
@@ -1,52 +1,76 @@
+    duration: '02:40:00',
+    lessons: '20',
+  },
+  {
+    title: 'Lorem',
     description:
       'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
-    language: 'English',
     duration: '02:40:00',
     lessons: '20',
-    link: 'link.io',
   },
   {
-    title:
-      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s',
+    title: 'Lorem',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+    duration: '02:40:00',
+    lessons: '20',
+  },
+  {
+    title: 'Lorem',
     description:
       'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
-    language: 'English',
     duration: '02:40:00',
     lessons: '20',
-    link: 'link.io',
   },
 ];
 
+export interface CourseStyleTypes {
+  fontSize?: number;
+  lineHeight?: number;
+  size?: string;
+  imgWidth?: number;
+  imgHeight?: number;
+}
+
+const styleProps: CourseStyleTypes = {
+  fontSize: 14,
+  lineHeight: 18,
+  size: 'medium',
+  imgWidth: 300,
+  imgHeight: 200,
```

Comment:
Move it please like primary or secondary style to the common component. Avoid sending a huge amount of props per tree. 
Confirm it with @hauteballe

Reply:
Replaced it with 1 prop "size" that has 3 options: small, medium, large. All the parts of Course component are styled according to the chosen size.